src/dotnet/commands/dotnet-build/CompileContext.cs

```diff
@@ -380,21 +384,18 @@ private bool InvokeCompileOnRootProject()
 
         private void CopyCompilationOutput()
         {
-            if (!string.IsNullOrEmpty(_args.OutputValue))
-            {
-                var calculator = _rootProject.GetOutputPaths(_args.ConfigValue, _args.BuildBasePathValue, _args.OutputValue);
-                var dest = calculator.RuntimeOutputPath;
-                var source = calculator.CompilationOutputPath;
-                foreach (var file in calculator.CompilationFiles.All())
+            var calculator = _rootProject.GetOutputPaths(_args.ConfigValue, _args.BuildBasePathValue, _args.OutputValue);
+            var dest = calculator.RuntimeOutputPath;
+            var source = calculator.CompilationOutputPath;
+            foreach (var file in calculator.CompilationFiles.All())
```

Comment:
This is the single usage of it everything else uses export with `CopyTo` extension available